<launch>
    <node name="usb_joystick" pkg="joy" type="joy_node" output="screen" required="false"
        clear_params="true">
        <param name="dev"
            value="/dev/input/js1"
            type="str" />
        <param name="deadzone" value="0.05" />
        <param name="autorepeat_rate" value="30.0" />
    </node>

    <node pkg="bw_teleop" type="tank_joystick_relay.py" name="tank_joystick_relay" output="screen"
        clear_params="true">
        <param name="axis_left" value="0" />
        <param name="axis_right" value="1" />
        <param name="scale_left" value="-3.0" />
        <param name="scale_right" value="3.0" />
        <param name="track_width" value="0.185" />
        <remap from="cmd_vel" to="cmd_vel/teleop" />
    </node>
</launch>